tests/error_message.rs

```diff
@@ -0,0 +1,17 @@
+extern crate snafu;
+
+use std::error::Error as _;
+use snafu::{Snafu, ResultExt};
+
+
+#[derive(Debug, Snafu)]
+enum Error {
+    BadId{ source: std::io::Error },
+}
+
+#[test]
+fn test_error_message() {
+    let err : Error = std::fs::read("/non/existent/file--test").eager_context(BadId{}).unwrap_err();
```

Comment:
For consistency, I'd call the context selector as `eager_context(BadId)`.

Reply:
Oh great, I didn't realize you could omit the brackets in this case.